example/src/main/scala/com/example/Access.scala

```diff
@@ -16,7 +16,7 @@ object Access {
     "sherlock.holmes" -> (Root.dev ++ securityAccess),
     "john.watson" -> (Production.dev ++ Staging.dev ++ Root.billing),
     "irene.adler" -> (Production.dev ++ Staging.dev ++ DataLake.dev),
-    "mycroft.holmes" -> (productionAccessLogs ++ securityCloudtrailLogs ++ serviceControlPolicyAccess)
+    "mycroft.holmes" -> (productionAccessLogs ++ securityCloudtrailLogs)
```

Comment:
Why is `serviceControlPolicyAccess` dropped?

Reply:
It doesn't exist, so was causing the project to fail to compile. Looking again I believe that this should actually refer to `BespokePolicies.serviceControlPolicyManagement` so have added that back in to include it in the example.